### ISSUE TITLE
Feathers doesn't exit when generating a Generic Service

### ISSUE THREAD

**Issue by silvestreh** [feathersjs/feathers-cli#35]
This is a low priority issue. After creating a `generic` service (no database) the CLI tool won't release the command prompt.

#### Steps to reproduce: 

1. Generate a new service using `feathers generate service`
2. When prompted, enter the name you want to give it, then select `generic` and, finally, select wether or not you need auth.
3. You'll see that Feathers generates the corresponding files and then the command prompt stays as `local@mbp~$` (or whatever your hostname is). Trying to run `ls` results in an Invalid command error and displays all available commands.

#### Workaround: 

Just hit <kbd>Ctrl</kbd>+<kbd>c</kbd> to exit manually.

**Comment by ekryski** [comment]
Fixed with version 2.0 that is coming. You can use the workaround for now.

**Comment by firrae** [comment]
To dig this up, I'm using the latest version of the cli but when I try to generate a new service I get the error as follows:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: No function expression found after module.exports
    at Object.exports.addLastInFunction (C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\generator-feathers\lib\transform.js:81:11)
    at importService (C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\generator-feathers\generators\service\index.js:16:15)
    at module.exports.generators.Base.extend.writing (C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\generator-feathers\generators\service\index.js:171:5)
    at Object.<anonymous> (C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\yeoman-generator\lib\base.js:439:23)
    at C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\run-async\index.js:26:25
    at new Promise (C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\core-js\modules\es6.promise.js:191:7)
    at C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\run-async\index.js:25:19
    at C:\Users\Steven.Lambe\AppData\Roaming\npm\node_modules\feathers-cli\node_modules\yeoman-generator\lib\base.js:440:9
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```
